packages/supabase/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "@plotwist/supabase",
```

Maybe rename to something more generic if this package is mean to be the database/server package and not just all things related to supabase.

```suggestion
  "name": "@plotwist/database",
```

```suggestion
  "name": "@plotwist/server",
```


```suggestion
  "name": "@plotwist/api",
```

---

I believe we can continue naming it `supabase`, as we may have other services de api or database at another time.

---

database is good